tests/test_dag_tree/test_create_input_structure.py

```diff
@@ -60,45 +61,49 @@ def functions_nested_and_duplicate_g() -> NestedFunctionDict:
             set(),
             {
                 "n1": {
-                    "a": None,
-                    "b": None,
+                    "a": "int",
+                    "b": "float",
+                    "c": "unknown_type",
```

Comment:
I am wondering (applies to the entire machinery, not just here) whether `"no_annotation_found"` would be clearer? In particular, if I am not initiated, I could think that `"unknown_type"` means "dags does not know this type".